examples/SED_BANDPASS_examples/example_SedBandpass_galaxy.py

```diff
@@ -87,7 +89,7 @@
 
 
 # show results
-print "#sedname      fluxnorm     redshift  ebv_gal   ebv_mw  magnitude "
+print("#sedname      fluxnorm     redshift  ebv_gal   ebv_mw  magnitude ")
 for i in range(len(galaxykeys)):
-    print "%s %.5g  %.3f %.5f %.5f %.5f" %(galaxykeys[i], fluxnorm[i], redshifts[i], ebv_gal[i], ebv_mw[i], mags[i])
+    print("%s %.5g  %.3f %.5f %.5f %.5f" %(galaxykeys[i], fluxnorm[i], redshifts[i], ebv_gal[i], ebv_mw[i], mags[i]))
     
```

Comment:
Oh.  Never mind.  I was trying to run it from the wrong directory.

I still don't want to touch these scripts if I don't have to.  They are, as I said, very old.